scripts/build.sh

```diff
@@ -45,7 +45,7 @@ cp -rf scripts/generate-certificate.sh $output_dir/install
 chmod +x $output_dir/install/genInventoryFile.py
```

Comment:
Do we need to quote everywhere`$output_dir` is used?

Reply:
output_dir is $(mktemp -d), which wont' have spaces . . unless TMPDIR is set and has spaces, for this PR, assuming it's not set to a dir with spaces